src/main/resources/Help/Templates/Article/WebHome.xml

```diff
@@ -53,50 +56,51 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed viverra enim quis t
 
-=== Sub-sub paragraph ===
+=== $services.localization.render('help.templates.subSubParagraph') ===
 
 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
 
 
-= Paragraph 2 =
+= $services.localization.render('help.templates.paragraph') 2=
 
 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
 
-== Sub-paragraph ==
+== $services.localization.render('help.templates.subParagraph') ==
 
 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
 
-== Sub-paragraph ==
+== $services.localization.render('help.templates.subParagraph') ==
 
 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
 )))
 
 
 (% class="col-xs-12 col-sm-4" %)
 (((
-{{box title="**Contents**"}}
+{{box title="**$services.localization.render('help.templates.contents')**"}}
 {{toc /}}
 {{/box}}
 
 [[image:Templates.Article.Template.WebHome@image1.jpg||style="width: 100%"]]
-//Figure 1: [[Sea&gt;&gt;https://commons.wikimedia.org/wiki/File:Isle_of_Icacos_II.jpg]]//
+//$services.localization.render('help.templates.figure') 1: [[$services.localization.render('help.templates.sea')&gt;&gt;https://commons.wikimedia.org/wiki/File:Isle_of_Icacos_II.jpg]]//
```

Comment:
Here it's just a demonstration of how you can put images. I don't think they need translations at all.

Reply:
Actually since the template itself is not translated, I'm not sure we should translate this page at all.